docs/development-process/standard-release-process.md

```diff
@@ -0,0 +1,60 @@
+# Standard Release Process
+
+The following document is a description of the standard release process of
+Zarhus-related images. Precise steps and differences from the standard
+process will be described in layer-specific documentation.
+
+## Process steps
+
+1. Make sure that everything that should go into the given release is merged
+   into `develop`.
+2. Bump `DISTRO_VERSION` in the `conf/distro/<distro-name>.conf` file.
+3. Fill up the `CHANGELOG.md` file with latest changes.
+    - Run `generate-changelog.sh` to generate the change notes for your release.
+    - Use the `generate-changed-recipes` script to print the list of changes in
+      recipe versions. The script uses build manifests - lists of recipes
+      contained in the image along with their versions. You must provide your
+      manifest and the current release's manifest as arguments to the script.
+      The latter can be found in the latest release's section in the GitHub
+      release pages. The current manifest can be found under the Yocto working
+      directory in
+      `build/tmp/deploy/images/{machine-name}/{image-name}.manifest`.
+    - Paste the script output into your release's section in `CHANGELOG.md`.
+      Additionally, you must explain why those changes were made and what they
+      introduce (unless the commit messages listed by `generate-changelog.sh`
+      already explain that).
+    - Describe any additional changes.
+4. Commit and push the changelog.
+5. Merge the changes from `develop` into `main`.
+5. Create and push tag that matches the newly bumped version with `v` added at
```

Comment:
@PLangowski, you could as well describe here where the tag should be pushed.

Reply:
cccce5c6d23977b4f5e7045fa772a1f1e6131056